src/contributing/releasing.md

```diff
@@ -134,6 +134,10 @@ This command will output the new changelog entry to STDOUT. Copy it and prepend
 existing `CHANGELOG.md` file, and make sure to change `HEAD` in the header of the entry
 to `$DEIS_RELEASE`.
 
+Also copy the component changelog to a global release changelog, organized by component.
```

Comment:
Can you specify where that changelog should live?

For example, should it just be a local file that the releaser should copy into a gist later?